sources/metrics/Stride.Metrics.ServerApp/Program.cs

```diff
@@ -0,0 +1,53 @@
+using System.Reflection;
+using Microsoft.EntityFrameworkCore;
+using Stride.Metrics.ServerApp.Data;
+using Stride.Metrics.ServerApp.Extensions;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add SQL Server database 
+builder.Services.AddDbContext<MetricDbContext>(opt =>
+{
+    opt.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"),
+    options => options.CommandTimeout(999999999));
```

Comment:
I would strongly suggest making this smaller 😄
15 minutes timeout should probably be enough - it's not like people will be waiting that long before getting discouraged anyways and maybe the query needs to be further optimized or db upgraded if it takes that long...

Reply:
Haha, yes, that was intended, just for testing purposes as some queries are very time consuming. I think I will remove it completely and try to cache some queries using views or additonal table

Reply:
Regarding the caching, from **.NET 7** you can cache very easily through minimal API, or a controller through `OutputCache`, maybe you know about that.

https://learn.microsoft.com/en-us/aspnet/core/performance/caching/output?view=aspnetcore-7.0

<img width="493" alt="image" src="https://github.com/stride3d/stride/assets/4528464/2af25f58-e914-4d63-83b3-08d234c0877a">